src/dev/flang/tools/fzjava/FZJava.java

```diff
@@ -86,6 +92,21 @@ public class FZJava extends Tool
   TreeMap<String, ForClass> _classes = new TreeMap<>();
 
 
+  /**
+   * An instance of FrontEnd, which is required to load library modules.
+   */
+  private FrontEnd _fe;
+
+
+  /**
+   * The features that have already been defined in the loaded library modules.
+   * Maps the qualified name of the feature to the qualified name of the feature.
+   * This is a map rather than a set because Java's Sets do not provide efficient
+   * methods for checking whether an element is in a set.
+   */
+  TreeMap<String, String> _existingFeatures = new TreeMap<String, String>();
```

Comment:
This is not true, `TreeSet.contains(x)` is just as efficient has `TreeMap.get(x) != null`.